VelocidiSDK/VelocidiSDK/Tracking/VSDKTrackingRequest.h

```diff
@@ -5,6 +5,9 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+  Class that wraps a NSURLMutableRequest with the necessary addons to make a tracking request to Velocidi's infrastructure
```

Comment:
Seems more accurate to me. What do you think?

(This suggestion also applies to other places)

```suggestion
  Class that wraps a NSURLMutableRequest with the necessary addons to make a tracking request to Velocidi's system
```